.github/workflows/release.yaml

```diff
@@ -98,6 +107,18 @@ jobs:
           git config user.email "noreply@cilium.io"
           git remote set-url origin https://x-access-token:${{ steps.get_token.outputs.app_token }}@github.com/${{ github.repository }}.git
 
+      - name: Install helm
```

Comment:
nit: Would be nice to move install steps like this to the beginning of the workflow so we process dependencies on external services (particularly cases like this that may be hosted outside GitHub) first.